tests/test_base_device.py

```diff
@@ -994,75 +902,20 @@ def test_estimator_with_various_multi_qubit_pauli_obs(
         sampler_execute.assert_not_called()
         estimator_execute.assert_called_once()
 
-        assert np.allclose(res, expectation, atol=0.3)  ## atol is high due to high variance
-
-    @pytest.mark.parametrize(
```

Comment:
we can't just update this test to be based on the new format? its nice to unit test that the process_estimator_job function returns the expected result

Reply:
This one I don't think is too challenging to update to the new format. There's no `EstimatorResult` equivalent for EstimatorV2 as far as I can tell but everything else should be updatable.

Reply:
Yeah ok I reworked this test but I didn't exactly care that much about `Type` of the data inside the result, since the focus of the test is on if process_estimator_job is working properly or not.